### DISCUSSION TITLE
我们在导入mysql业务数据遇到的不方便的地方

### DISCUSSION THREAD

**Discussion by gochiel** [apache/doris#31500] [RFC]
Doris有没有计划出个类似 ClickHouse 的 MaterializedMySQL 引擎，可以极大方便直接同步mysql的业务数据而不用再引入第三方的架构，这块对于数据同步的提升还是非常大的，简化了etl。这也是我们公司在选型ck还是doris，一直卡着的点

**Comment by ihadoop** [comment]
我看在2024的路线图中有这个功能
 Zero-ETL: Built-in data integration from OLTP CDC to Doris
https://github.com/apache/doris/issues/30669

**Reply by gochiel** [reply]
感谢，看上去这个功能有在计划里，这个功能对于业务数据库的etl工作帮助还是非常大的
